bluemira/codes/process/_setup.py

```diff
@@ -104,14 +104,14 @@ def _write_in_dat(self, use_bp_inputs: bool = True):
         # Load defaults in bluemira folder
         writer = _make_writer(self.template_in_dat)
 
-        if use_bp_inputs:
-            inputs = self._get_new_inputs(remapper=update_obsolete_vars)
-            for key, value in inputs.items():
-                writer.add_parameter(key, value)
-            for key, value in self.problem_settings.items():
-                writer.add_parameter(key, value)
+        # if use_bp_inputs:
+        #     inputs = self._get_new_inputs(remapper=update_obsolete_vars)
+        #     for key, value in inputs.items():
+        #         writer.add_parameter(key, value)
+        #     for key, value in self.problem_settings.items():
+        #         writer.add_parameter(key, value)
```

Comment:
@je-cook I needed to do this to play around, as discussed, but results in errors after PROCESS run in EU-DEMO.

Reply:
no problem, I'll rebase mine and see if I can find the root.